src/EA.Weee.Integration.Tests/Handlers/DirectRegistrant/CompleteOrganisationTransactionHandlerIntegrationTests.cs

```diff
@@ -32,10 +32,14 @@ public class WhenICompleteAnRegisteredCompanyOrganisationTransaction : CompleteO
                 var registeredCompanyDetails = fixture.Build<OrganisationViewModel>()
                     .With(r => r.Address, addressData).Create();
 
```

Comment:
@will-hope98  should create a new integration test in here and validate that when you set the ProducerRegistrationNumber it is set on on the direct registrant.